org.learningequality.Kolibri.yaml

```diff
@@ -15,13 +15,19 @@ finish-args:
   - --socket=x11
   - --system-talk-name=org.learningequality.Kolibri.Daemon
   - --env=KOLIBRI_HOME=~/.var/app/org.learningequality.Kolibri/data/kolibri
+  - --env=PYTHONPATH=/app/kolibri-plugins/lib/python
 
 add-extensions:
   org.learningequality.Kolibri.Content:
     version: '1.0'
     directory: share/kolibri-content
     subdirectories: true
     no-autodownload: true
+  org.learningequality.Kolibri.Plugin:
```

Comment:
How forwards- and backwards-compatible are plugins versus the core?

- Will Kolibri of today work with the Explore plugin of 12 months from now?
- Will the Explore plugin of today work with Kolibri of 12 months from now?

If no, this extension point should probably be versioned with e.g. the Kolibri major version or whatever the "plugin-api-stable" guarantee is. Flatpak will typically try to update apps' extensions at the same time as the app but this is not guaranteed.

Reply:
Indeed. The 2 versions are tightly coupled. The current kolibri-explore-plugin is known to work for Kolibri v0.14.6 only.